sdks/go/pkg/beam/core/metrics/store.go

```diff
@@ -155,6 +155,7 @@ type ptCounterSet struct {
 type bundleProcState int
 
 const (
+	// Bundle execution states
```

Comment:
Technically, you'll need to add a comment that starts with  `// StartBundle` and so forth here.